samples/Client/DaprClient/Program.cs

```diff
@@ -209,6 +212,34 @@ internal static async Task InvokeBalanceServiceOperationAsync(DaprClient client)
             Console.WriteLine($"Received balance {res.Balance}");
         }
 
+        internal static async Task InvokeThrowExceptionOperationAsync(DaprClient client)
+        {
+            Console.WriteLine("Invoking ThrowException");
+            var data = new { id = "17", amount = (decimal)10, };
+
+            HTTPExtension httpExtension = new HTTPExtension()
+            {
+                Verb = HTTPVerb.Post
+            };
+
+            try
+            {
+                // Invokes a POST method named "Withdraw" that takes input of type "Transaction" as define in the RoutingSample.            
+                await client.InvokeMethodAsync<object>("controller", "throwException", data, httpExtension);
+            }
+            catch (Grpc.Core.RpcException ex)
+            {
+                var entry = ex.Trailers.Get("grpc-status-details-bin");
+                var status = Google.Rpc.Status.Parser.ParseFrom(entry.ValueBytes);
+                Console.WriteLine("Grpc Exception Message: " + status.Message);
+                Console.WriteLine("Grpc Statuscode: " + status.Code);
+                var details = status.Details[0];
+                var rpcError = details.Unpack<Google.Rpc.ErrorInfo>();
+                Console.WriteLine("Grpc Exception: Http Error Message: " + rpcError.Metadata["http.error_message"]);
+                Console.WriteLine("Grpc Exception: Http Status Code: " + rpcError.Metadata["http.code"]);
```

Comment:
Are there string constants defined for these anywhere? I guess what I'm really asking about is whether `http.code` is a Dapr specific convention or a gRPC convention?

Reply:
this is dapr-specific convention. GRPC doc doesn't provide guideline for metadata in ErrorInfo.

Reply:
http.code and http.error_message are dapr-specific - https://github.com/dapr/dapr/blob/bdd465455c8c6c2b00b3a7e1d32f5f5757b75d53/pkg/messaging/v1/util.go#L56-L57

Reply:
fixed